database/database.go

```diff
@@ -27,6 +27,9 @@ func New(cfg *config.Config, interrupt <-chan struct{}) (model.DataBase, error)
 }
 
 func Cleanup(cfg *config.Config) {
+	if len(cfg.DataDir) <= 0 {
+		return
```

Comment:
If not providing ```cfg.dataDir``` means using an in-memory database, it is better to add a line of comments.
```golang 
return // do nothing when using the memory db
```

Reply:
👌

> If not providing `cfg.dataDir` means using an in-memory database, it is better to add a line of comments.
> 
> ```go
> return // do nothing when using the memory db
> ```

👌